### ISSUE TITLE
`this` not properly highlighted

### ISSUE THREAD

**Issue by escKeyStroke** [palantir/tslint#4525]
### Bug Report

- __TSLint version__: 1.0.0
- __TypeScript version__: 3.3.1
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
class C {
  
  f: (g: Function) => any 
  = g => g();

  h: () => string 
  = () =>'a';

  k: (g: Function) => any 
  =
  this.f
  (
    this.h
  )

}
```

with `tslint.json` configuration: default

#### Actual behavior
`this` is not highlighted as in other parts of the code
#### Expected behavior
This is valid code, thus `this` should be highlighted

I won't go into the details of why I'm styling my code like this:
```ts
  this.f
  (
    this.h
  )
```
instead of like this (this second case works)
```ts
  this.f  (
    this.h
  )
```
What matters is that the first case is valid code not properly syntax-highlighted.

**Comment by JoshuaKGoldberg** [comment]
Hi @escKeyStroke! This repository is for TSLint, which only runs extra analysis (linting) on TypeScript code. TSLint doesn't handle syntax highlighting.

I think you'll want to file this issue on [VS Code](https://github.com/microsoft/vscode). Best of luck!

**Comment by escKeyStroke** [comment]
Thanks @JoshuaKGoldberg. Apologies for the waste of time. https://github.com/Microsoft/vscode/issues/69281
